docs/core/compatibility/extensions/6.0/bind-single-elements-to-array.md

```diff
@@ -0,0 +1,108 @@
+---
+title: "Breaking change: Configuration binder binds single elements to an array"
+description: Learn about the .NET 6.0 breaking change in .NET extensions where Microsoft.Extensions.Configuration binder will now bind a single element to an array.
+ms.date: 09/08/2021
+---
+# Microsoft.Extensions.Configuration binder binds single elements to an array
+
+Starting in .NET 6, the <xref:Microsoft.Extensions.Configuration?displayProperty=fullName> binder will bind single elements to an array.
+
+## Version introduced
+
+6.0 RC 1
+
+## Previous behavior
+
+Consider the following JSON snippet and C# class definitions:
+
+```json
+{
+   "tvshow": {
+        "metadata": {
+            "name": "PrisonBreak"
+          }
+    }
+}
+```
+
+```csharp
+public class TvShow
+{
+  public Metadata[] Metadata { get; set; }
+}
+
+public class Metadata
+{
+  public string Name { get; set; }
+}
+```
+
+The JSON configuration provider would not bind that JSON because it does not specify an array. The JSON would need to appear as follows in order to bind:
```

Comment:
I'm not sure if I explained this correctly. Please correct if necessary. I'm trying to show what's change for the JSON provider.